Signal/src/view controllers/AdvancedSettingsTableViewController.m

```diff
@@ -52,18 +54,27 @@ - (void)loadView {
     [super loadView];
 
     self.title = NSLocalizedString(@"SETTINGS_ADVANCED_TITLE", @"");
-
+    
+    // WebRTC
+    self.enableWebRTCCell                        = [[UITableViewCell alloc] init];
```

Comment:
It should pick up the .clang-format in the containing directory. Nothing that you should have to configure. I think it has to be staged but not committed to work.

I've added it to my .git/hooks/pre-commit

So my workflow looks like:

```
$ edit stuff
$ git add
$ git clang-format
$ git add # again
$ git commit -m "foo"
```

And if I forgot to run git-format before commiting, the precommit hook will sully my working directory as a reminder. At which point I `git add` and `git commit --amend` the formatting changes before pushing.